src/components/navbar/index.js

```diff
@@ -2,42 +2,76 @@ import React from 'react';
 import styled from 'styled-components';
 
 const NavWrapper = styled.header`
```

Comment:
Probably not good practice to have this as the header-- you could replace header with nav, remove the nav tags from your component, and take away the inner nav styling here